bigtop-packages/src/common/tez/install_tez.sh

```diff
@@ -109,7 +109,10 @@ install -d -m 0755 $PREFIX/$DOC_DIR
 install -d -m 0755 $PREFIX/$CONF_DIR
 install -d -m 0755 $PREFIX/$MAN_DIR
 
-cp tez-site.xml  $PREFIX/$CONF_DIR
+tar -C $PREFIX/$LIB_DIR -xzf $BUILD_DIR/tez-dist/target/tez*-minimal.tar.gz
 
-tar -C  $PREFIX/$LIB_DIR -xzf  $BUILD_DIR/tez-dist/target/tez*-minimal.tar.gz
+cp tez-site.xml $PREFIX/$CONF_DIR
+
+TEZ_TAR=`find $BUILD_DIR/tez-dist/target/ -name "tez-*.tar.gz" | grep -v "minimal"`
```

Comment:
Using glob pattern could be a bit safer if other tez-*.tar.gz appears in the future.

```
$ ls tez-dist/target/
archive-tmp  maven-archiver  maven-shared-archive-resources  tez-0.10.1  tez-0.10.1-minimal  tez-0.10.1-minimal.tar.gz  tez-0.10.1.tar.gz  tez-dist-0.10.1-tests.jar
$ export TEZ_TAR=tez-dist/target/tez-[[:digit:]]*[[:digit:]].tar.gz
$ echo $TEZ_TAR
tez-dist/target/tez-0.10.1.tar.gz
```

Reply:
Done